indexd/index/drivers/alchemy.py

```diff
@@ -171,21 +191,26 @@ def hashes_to_urls(self, size, hashes, start=0, limit=100):
 
             return [r.url for r in query]
 
-    def add(self, form, size=None, urls=[], hashes={}, did=None):
+
+    def add(self, form, size = None, urls = [], hashes = {}, did = None, baseid = None):
```

Comment:
keyword arguments should not have space around `=`; so e.g. `size=None` instead of `size = None`
(here and elsewhere)